packages/plugin-loansresearch-ui/src/containers/Settings.tsx

```
@@ -0,0 +1,70 @@
+export default withProps<Props>(
+  compose(
+    graphql<Props, ConfigsQueryResponse>(gql(queries.adConfigs), {
+      name: 'configsQuery',
+      options: () => ({
+        variables: {
+          code: 'ACTIVEDIRECTOR',
+        },
+        fetchPolicy: 'network-only',
+      }),
+    }),
+    graphql<{}>(gql(mutations.adConfigUpdate), {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)